File: FoilCLI/IMarshaller.cs
using System;
using System.IO;

namespace FoilForDotNet
{
	/// <summary>
	/// Summary description for IMarshaller.
	/// </summary>
	public interface IMarshaller
	{
		public	void marshall(Object o,TextWriter w, IMarshaller baseMarshaller, int flags,int depth);
	}
}

File: com/richhickey/foil/IMarshaller.java
/*
 * Created on Dec 7, 2004
 *
 *   Copyright (c) Rich Hickey. All rights reserved.
 *   The use and distribution terms for this software are covered by the
 *   Common Public License 1.0 (http://opensource.org/licenses/cpl.php)
 *   which can be found in the file CPL.TXT at the root of this distribution.
 *   By using this software in any fashion, you are agreeing to be bound by
 * 	 the terms of this license.
 *   You must not remove this notice, or any other, from this software.
 */
package com.richhickey.foil;

import java.io.Writer;

/**
 * @author Rich
 *
 * TODO To change the template for this generated type comment go to
 * Window - Preferences - Java - Code Style - Code Templates
 */
public interface IMarshaller {
	void marshall(Object o,Writer w, IMarshaller baseMarshaller, int flags,int depth);

}
